rpc/src/rpc_subscriptions.rs

```diff
@@ -210,11 +214,16 @@ impl RecentItems {
                 .expect("total bytes underflow");
         }
 
-        datapoint_info!(
-            "rpc_subscriptions_recent_items",
-            ("num", self.queue.len(), i64),
-            ("total_bytes", self.total_bytes, i64),
-        );
+        let now = Instant::now();
+        let last_metrics_ago = now.duration_since(self.last_metrics_submission);
+        if last_metrics_ago > RPC_NOTIFICATIONS_METRICS_SUBMISSION_INTERVAL_MS {
+            datapoint_info!(
+                "rpc_subscriptions_recent_items",
+                ("num", self.queue.len(), i64),
+                ("total_bytes", self.total_bytes, i64),
+            );
+            self.last_metrics_submission = now;
+        }
```

Comment:
Do you mind also adding a trace log such that a node operator could turn on to see all the movements of the queue length and total_bytes, if needed?

Reply:
done!
https://github.com/solana-labs/solana/blob/c79262268356f1ace3758c94309099d93d0fc144/rpc/src/rpc_subscriptions.rs#L226-L232